### PR TITLE
Import gripper glue

### PR DESCRIPTION
**Pull request by cdussieux**
Cherry-picked bundle of commit that build glue_gripper plugin to allow reachy to grasp in simulation

### CODE REVIEW

File: reachy_gazebo_gripper_glue/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+As an argument, glue_threshold_distance can be specified (default value: 0.1)
+```commandline
+<plugin name="reachy_gazebo_gripper_glue" filename="libreachy_gazebo_gripper_glue.so">
+<glue_threshold_distance>0.4</glue_threshold_distance>
+</plugin>
+```
+
+
+il y a un petit comment `// LOOK AT ME, I'M THE GLUE NOW` pour indiquer le principal levier de mofi à savoir la condition de trigger du glue
```
Comment (simheo) [inline reachy_gazebo_gripper_glue/ReadMe.md:33]: Principal levier de mofi ?

Comment (pierre-rouanet) [inline reachy_gazebo_gripper_glue/ReadMe.md:33]: Excuse my French :)

File: reachy_gazebo_gripper_glue/src/reachy_gazebo_gripper_glue.cpp [hunk 2]
```diff
@@ -0,0 +1,170 @@
+#include <gazebo/gazebo.hh>
+#include <gazebo/physics/physics.hh>
+
+/*
+ * I'm glue da ba dee da ba daa
```
Comment (pierre-rouanet) [inline reachy_gazebo_gripper_glue/src/reachy_gazebo_gripper_glue.cpp:5]: Damn you...

**Review by pierre-rouanet — DISMISSED** [review]
LGTM, except the minor comment.
I've not tested on a real setup though.

**Review by simheo — APPROVED** [review]

**Pull request merged**